CMakeLists.txt

```diff
@@ -56,6 +56,9 @@ if(OS_WINDOWS)
   configure_file(cmake/bundle/windows/installer-Windows.iss.in
                  ${CMAKE_BINARY_DIR}/installer-Windows.generated.iss)
 
+  configure_file(cmake/bundle/windows/resource.rc.in ${CMAKE_PROJECT_NAME}.rc)
```

Comment:
I'd probably just add `${CMAKE_BINARY_DIR}` here to not pollute the checkout outside the build directory too much.